ElrondNetwork/elrond-go
# En 7325 log stats files update

- removed `stats.txt` file, data is being written in log files directly
- added the possibility to use multiple log files instead of just one
- added `Nonce` field in heartbeat structure

Manual testing procesdures:
- The file `stats.txt` should no longer appear, its contents should appear in the log files, search string: `node statistics`
- Running the node with the config option `LogFileLifeSpanInSec = 10` in `config.toml` file will generate a new log file each 10 seconds
- Use mainnet configs, try to sync the node with the rest of the network. At this current moment, the network will use the old heartbeat DTO version (no `Nonce` field). All received heartbeat messages should be valid, node should be able to sync, `Nonce` values should be 0
- Running the local testnets should yield the current nonces on all connected nodes.
